recipes/redis-plus-plus/all/conanfile.py

```diff
@@ -64,6 +64,13 @@ def _min_compilers_version(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+
+    def validate(self):
```

Comment:
keep validate() after requirements() please, to be able to read and understand the recipe from top to bottom.

Reply:
Is it possible to define a validation hook for it?